ambry-api/src/main/java/com.github.ambry/clustermap/ClusterMap.java

```diff
@@ -37,6 +37,12 @@
   List<? extends PartitionId> getWritablePartitionIds();
 
   /**
+   * Gets a list of all partitions in the cluster
+   * @return a list of all partitions in the cluster
+   */
+  List<? extends PartitionId> getAllPartitions();
```

Comment:
I would like to have it in lines with other methods here. Can you change it in the PR you plan to put up.